automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -1,4 +1,4 @@
-package org.broadinstitute.dsde.firecloud.test.api.orch
+package org.broadinstitute.dsde.test.api.orch
```

Comment:
This is a fix because IntelliJ was warning that the package didn't match the directory structure.